Gems/WhiteBox/Code/Source/WhiteBoxComponent.cpp

```diff
@@ -42,7 +42,7 @@ namespace WhiteBox
 
         // generate the mesh
         // TODO: LYN-786
```

Comment:
update: Remove LYN reference (we can make a GHI to track this if that would still be useful)

Reply:
Fixed 👍